libs/pages/settings/src/lib/ui/page-organization-github-repository-access/page-organization-github-repository-access.tsx

```diff
@@ -25,65 +31,98 @@ export interface PageOrganizationGithubRepositoryAccessProps {
 }
 
 export function PageOrganizationGithubRepositoryAccess(props: PageOrganizationGithubRepositoryAccessProps) {
+  const { openModal, closeModal } = useModal()
+  const { organizationId = '' } = useParams()
+
   return (
-    <div className="flex flex-col justify-between w-full">
+    <Section className="flex flex-col justify-between w-full">
       <div className="p-8 max-w-content-with-navigation-left">
         <div className="flex justify-between mb-8">
-          <div>
+          <div className="mr-5">
+            {/* TODO: Fix need to use <Heading /> component */}
             <h1 className="h5 text-neutral-400 mb-2">Github Repository Access</h1>
```

Comment:
I can't use `<Heading />` because it makes inconsistency size between all settings pages